### PR TITLE
Add a way to run bootnode under systemd

### PR DESCRIPTION
**Pull request by jakubgs**
After reviewing some stuff for https://github.com/status-im/design/issues/40 I decided to refactor the way we start a mailserver under systemd. I've made it so it works the same as the container ones, now they also use `Makefile`s.

In addition I've implemented the same thing for running Status bootnodes. But in order to streamline that process I've also implemented the `-genkey` and `-writeaddress` flags for `bootnode` service.

### CODE REVIEW

File: cmd/bootnode/main.go [hunk 1]
```diff
@@ -69,6 +83,10 @@ func main() {
 			log.Crit("unable to convert decoded hex into ecdsa.PrivateKey", "data", key, "error", err)
 		}
 	}
+	if *writeAddr {
+		fmt.Printf("%x\n", crypto.FromECDSAPub(&nodeKey.PublicKey)[1:])
```
Comment (adambabik) [inline cmd/bootnode/main.go:88]: Why omitting the first byte? It's that compressed/uncompressed info?

Comment (jakubgs) [reply cmd/bootnode/main.go:88]: I just copied it from `go-ethereum` repo:
https://github.com/ethereum/go-ethereum/blob/c2d65d34/cmd/bootnode/main.go#L90-L93
Any idea why they did it like that?

Comment (jakubgs) [reply cmd/bootnode/main.go:88]: Oh, I think they are just removing the `0x` from it so it's ready to be used in an `enode://` address.

Comment (adambabik) [reply cmd/bootnode/main.go:88]: No, it's a bytes slice. `0x` is added when formatting to a string. It's about that uncompressed byte: https://github.com/golang/go/blob/go1.13.1/src/crypto/elliptic/elliptic.go#L310. LGTM but we can add a comment why we do `[1:]`. I assume this byte is omitted in enode ID...?

Comment (jakubgs) [reply cmd/bootnode/main.go:88]: Okay, I added a comment explaining it. Okay to merge?

**Review by adambabik — APPROVED** [review]

**Pull request merged**